scalableminds/webknossos
# Show active users modal

### Mailable description of changes:
 - Admins now can see and transfer all active tasks of a project to a single user

### URL of deployed dev instance (used for testing):
- 

### Steps to test:
- go to the admin -> projects
- use the show active users button on the right side to open the modal
- check the table if its showing all users with their active tasks
- select a user and then transfer the active tasks to the user (a toast should show up to confirm the transfer)
- then check the tasks-tab from the dashboard to confirm the transfer

### Issues:
- fixes #2670
- fixes #2862

------
